### ISSUE TITLE
build failure while installing prc-tools

### ISSUE THREAD

**Issue by chainq** [jichu4n/homebrew-palm-os#1]
I recently tried to install prc-tools from homebrew using your tap and the build fails with:

```
==> Installing prc-tools from jichu4n/palm-os
==> Downloading https://github.com/jichu4n/prc-tools-remix/archive/2.3.3.tar.gz
==> Downloading from https://codeload.github.com/jichu4n/prc-tools-remix/tar.gz/2.3.3
######################################################################## 100.0%
==> ../prc-tools-2.3/configure --enable-targets=m68k-palmos,arm-palmos --enable-languages=c,c++ --disable-nls --host=i686-apple-darwin --with-palmdev-prefix=/usr/local/opt/p
==> make
Last 15 lines from /Users/charlie/Library/Logs/Homebrew/prc-tools/02.make:
clang -c   -g -DIN_GCC -DCROSS_COMPILE  -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes    -DHAVE_CONFIG_H    -I. -Icp -I../../../../prc-tools-2.3/gcc/gcc -I../../../../prc-tools-2.3/gcc/gcc/cp -I../../../../prc-tools-2.3/gcc/gcc/config -I../../../../prc-tools-2.3/gcc/gcc/../include ../../../../prc-tools-2.3/gcc/gcc/cp/dump.c -o cp/dump.o
clang -c   -g -DIN_GCC -DCROSS_COMPILE  -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes    -DHAVE_CONFIG_H    -I. -Icp -I../../../../prc-tools-2.3/gcc/gcc -I../../../../prc-tools-2.3/gcc/gcc/cp -I../../../../prc-tools-2.3/gcc/gcc/config -I../../../../prc-tools-2.3/gcc/gcc/../include ../../../../prc-tools-2.3/gcc/gcc/cp/optimize.c -o cp/optimize.o
clang -c   -g -DIN_GCC -DCROSS_COMPILE  -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes    -DHAVE_CONFIG_H    -I. -Icp -I../../../../prc-tools-2.3/gcc/gcc -I../../../../prc-tools-2.3/gcc/gcc/cp -I../../../../prc-tools-2.3/gcc/gcc/config -I../../../../prc-tools-2.3/gcc/gcc/../include ../../../../prc-tools-2.3/gcc/gcc/cp/mangle.c -o cp/mangle.o
clang -c   -g -DIN_GCC -DCROSS_COMPILE  -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes    -DHAVE_CONFIG_H    -I. -Icp -I../../../../prc-tools-2.3/gcc/gcc -I../../../../prc-tools-2.3/gcc/gcc/cp -I../../../../prc-tools-2.3/gcc/gcc/config -I../../../../prc-tools-2.3/gcc/gcc/../include ../../../../prc-tools-2.3/gcc/gcc/cp/cp-lang.c -o cp/cp-lang.o
clang   -g -DIN_GCC -DCROSS_COMPILE  -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wtraditional -pedantic -Wno-long-long   -DHAVE_CONFIG_H  -o cc1plus \
	      cp/call.o cp/decl.o cp/expr.o cp/pt.o cp/typeck2.o cp/class.o cp/decl2.o cp/error.o cp/lex.o cp/parse.o cp/ptree.o cp/rtti.o cp/spew.o cp/typeck.o cp/cvt.o cp/except.o cp/friend.o cp/init.o cp/method.o cp/search.o cp/semantics.o cp/tree.o cp/repo.o cp/dump.o cp/optimize.o cp/mangle.o cp/cp-lang.o attribs.o c-common.o c-format.o c-pragma.o c-semantics.o c-lex.o c-dump.o  c-pretty-print.o c-opts.o main.o libbackend.a libcpp.a   ../libiberty/libiberty.a
Undefined symbols for architecture x86_64:
  "_libc_name_p", referenced from:
      _nothrow_libfn_p in except.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [cc1plus] Error 1
make[2]: *** [all-gcc] Error 2
make[1]: *** [real-gcc] Error 2
make: *** [arm-palmos] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/jichu4n/homebrew-palm-os/issues
```

I followed the instructions in the README in prc-tools-remix Github repo.

My Homebrew version is:

```
Homebrew 1.3.1
Homebrew/homebrew-core (git revision 00605; last commit 2017-08-13)
```

My clang version is:
```
Apple LLVM version 8.1.0 (clang-802.0.42)
Target: x86_64-apple-darwin16.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

I run macOS 10.12.5.

**Comment by jamis** [comment]
I the same error as well, on macOS 10.14.2.

~~~
Homebrew 1.8.6
Homebrew/homebrew-core (git revision 8d0d13; last commit 2018-12-21)
~~~

And:

~~~
$ clang --version
Apple LLVM version 10.0.0 (clang-1000.11.45.5)
Target: x86_64-apple-darwin18.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
~~~

**Comment by jamis** [comment]
FWIW, applying the following PR fixed the problem for me:

https://github.com/jichu4n/prc-tools-remix/pull/1

If that could get applied, it ought to fix the problem here, when installing via homebrew.

**Comment by jichu4n** [comment]
Merged PR jichu4n/prc-tools-remix#1. Thanks all!
